restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/AttributeValidator.java

```diff
@@ -0,0 +1,548 @@
+package org.restcomm.connect.interpreter.rcml;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+
+public interface AttributeValidator {
+
+	public boolean RecordValidator (Tag tag);

```

Comment:
i expected to have a generic interface API, and different validator impl depending on the RCML verb to validate. Instead we have a particular method to validate different verbs. What is the benefit of defining an interface with such API?